### PR TITLE
Android solutions with RMS

### PR DESCRIPTION
**Pull request by m0bilesecurity**
Solving OWASP UnCrackable Android App Level 1 and 2 with Runtime Mobile Security (RMS) - VIDEOs

### CODE REVIEW

File: Crackmes/README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ This app is compatible with Android 4.4 and up.
 - [Solution using jdb](../Document/0x05c-Reverse-Engineering-and-Tampering.md#debugging-with-jdb "Solution using jdb")
 - [Solution using Frida by Eduardo Novella](https://enovella.github.io/android/reverse/2017/05/18/android-owasp-crackmes-level-1.html "Solution by Eduardo Novella")
 - [Solution using Xposed by sh3llc0d3r](http://sh3llc0d3r.com/owasp-uncrackable-android-level1/ "Solution by sh3llc0d3r")
+- [Solution using RMS by @mobilesecurity_ (VIDEO)](https://youtu.be/P6rNPkM2DdY "Solution by @mobilesecurity_")
```
Comment (sushi2k) [inline Crackmes/README.md:32]:
```suggestion
- [Solution using RMS by @mobilesecurity_ (video)](https://youtu.be/P6rNPkM2DdY "Solution by @mobilesecurity_")
```

File: Crackmes/README.md [hunk 2]
```diff
@@ -52,6 +53,7 @@ This app is compatible with Android 4.4 and up.
 - [Solution using Frida and radare2 by c0dmtr1x](https://www.codemetrix.net/hacking-android-apps-with-frida-3/ "Solution by c0dmtr1x")
 - [Solution using Frida by Eduardo Novella](https://enovella.github.io/android/reverse/2017/05/20/android-owasp-crackmes-level-2.html "Solution by Eduardo Novella")
 - [Solution using patches by sh3llc0d3r](http://sh3llc0d3r.com/owasp-uncrackable-android-level2/ "Solution by sh3llc0d3r")
+- [Solution using RMS by @mobilesecurity_ (VIDEO)](https://youtu.be/xRQVljerl0A "Solution by @mobilesecurity_")
```
Comment (sushi2k) [inline Crackmes/README.md:56]:
```suggestion
- [Solution using RMS by @mobilesecurity_ (video)](https://youtu.be/xRQVljerl0A "Solution by @mobilesecurity_")
```

**Review by sushi2k — COMMENTED** [review]
Thanks @m0bilesecurity!

**Review by sushi2k — APPROVED** [review]
I changed "VIDEO" to "video". Otherwise, thank you very much for sharing the video!

**Comment by m0bilesecurity** [comment]
many thanks @sushi2k 🙏🏻 🙏🏻 🙏🏻

**Pull request merged**